priv/repo/migrations/20190418150514_add_slug_to_podcast_table.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Radiator.Repo.Migrations.AddSlugToPodcastTable do
+  use Ecto.Migration
+
+  def change do
+    alter table(:podcasts) do
+      add :slug, :string
```

Comment:
Feel free to add this to the original podcast creation migration instead of its own migration. We recently decided to adjust existing migrations for now instead of adding new ones.